### PR TITLE
better lookup for _schema_name

### PR DESCRIPTION
**Pull request by domdinicola**
try to find "_schema_name" both in task.request.headers and request.task
address https://github.com/maciej-gol/tenant-schemas-celery/issues/46

### CODE REVIEW

**Comment by maciej-gol** [comment]
Hey! Thanks for the patch! What issue in particular does it solve?

sob., 19 gru 2020, 17:28 użytkownik Domenico <notifications@github.com>
napisał:

**Comment by domdinicola** [comment]
Hi @maciej-gol ,
This is the issue.
https://github.com/maciej-gol/tenant-schemas-celery/issues/46

Basically I find the request.task.header set (with sentry headers), so it will ignore request.task, where my "_schema_name" key is.

**Comment by maciej-gol** [comment]
What celery version you are using?

niedz., 20 gru 2020, 17:17 użytkownik Domenico <notifications@github.com>
napisał:

**Comment by domdinicola** [comment]
celery==4.3.0
django-celery-beat==1.5.0
django-celery-email==2.0.2
django-celery-results==1.1.2
tenant-schemas-celery==0.2.1
sentry-sdk==0.19.1

**Comment by maciej-gol** [comment]
Thanks for the report! Im currently away from my personal computer, and
will be able to look into early January. Unfortunately.

niedz., 20 gru 2020, 21:36 użytkownik Domenico <notifications@github.com>
napisał:

**Comment by domdinicola** [comment]
no problem, let's look at it in Jan! Enjoy your holidays :)

File: tenant_schemas_celery/app.py [hunk 1]
```diff
@@ -18,8 +18,9 @@ def get_schema_name_from_task(task, kwargs):
         return kwargs.pop("_schema_name", None)
 
     # In some cases (like Redis broker) headers are merged with `task.request`.
-    task_headers = task.request.headers or task.request
-    return task_headers.get("_schema_name")
+    if "_schema_name" in task.request.headers:
```
Comment (maciej-gol) [inline tenant_schemas_celery/app.py:21]: It's good to check whether the `headers` attr is set. Previous expression handled such cases.

```suggestion
    if task.request.headers and "_schema_name" in task.request.headers:
```

**Comment by maciej-gol** [comment]
Also, please rebase on the newest `master`. Includes migrated CI from Travis to Github Actions and fixes for Celery 5.0

**Comment by domdinicola** [comment]
> Also, please rebase on the newest `master`. Includes migrated CI from Travis to Github Actions and fixes for Celery 5.0

addressed!

**Comment by maciej-gol** [comment]
Thanks! Should release a new version soon, most likely today.

**Comment by domdinicola** [comment]
> Thanks! Should release a new version soon, most likely today.

much appreciated!

**Pull request merged**